### ISSUE TITLE
Beacon have less loot than I have set...

### ISSUE THREAD

**Issue by Tek4s** [thommcgrath/Beacon#99]
#Hi,
I wanted to change the complete loot of all beacons, but unfortunately despite the settings (example: min.Items: 7 Max Items: 7. In the set are 7 items) only 1-2 items per beacon and that with all beacons (White- Red). He varies the sets but I just can not find the problem ...
can someone help me?

all beacon  Configs
https://www.file-upload.net/download-13259180/Codes.ini.html

The Settings from the Beacon app
https://www.file-upload.net/download-13259181/1.JPG.html

**Comment by thommcgrath** [comment]
Click the pencil icon at the bottom of the loot sources list to edit the drop, you need to change the min and max item set numbers.

**Comment by thommcgrath** [comment]
No, what you’ve described is exactly how it works. Per that screenshot, if you have the min and max item sets set to one, it’ll pick one set. If it picks the set shown in the screenshot, it would pick all 7 lines.

**Comment by thommcgrath** [comment]
Also, I removed the links to the files in the original comment because the host frequently redirects to malware sites instead of the desired file.

**Comment by Tek4s** [comment]
okay i'm sorry i did not know that this site spread malware ...
I would like to say that the beacon chooses 1 out of 6 sets at random and the 7 items are always taken in the sets. But I always get a different number of items mostly 1-2

**Comment by thommcgrath** [comment]
So, I’m that case... I’m not sure. Try posting the .beacon file here so I can look at it. You can simply drag files into the comment field to upload them. It’s possible a mod is to blame, but I’m not convinced that is the issue. I’ve never seen the game fail to respect the settings produced, without it being a mod issue though.

**Comment by Tek4s** [comment]
okay no problem, thanks
I do not know if it is helpful but here is the list of all the mods I use

715894094,           Elixir of Hunger           
743217220,           Extra Large Storage Box    
793605978,           Super Spyglass             
1440414363 ,        All Dinos Allow Guns     
761535755,           Ultra Stacks               
1251726467,         Zoa´s Potions          
1382346085,         SuperFastGrapplingHook    
1449245048,         Narcotics & Tranq-Arrows  
1399900808,         WATER BLOCKS            
558079412,           ACM V2.7                   
1383175081,         ZeroGSpeedGlider      
566887000,           Pet Finder                  
566885854,           Death Helper              
1171853632,         Meat Spoiler             
889745138,           Awesome Teleporters!      
895711211,           Classic Flyers           
1341173952,         Dino Storage                 
510590313,           GlassMetal                  
1133313678,         Awesome Ammo Mod        
821530042,           Upgrade Station   


[Speed Ark PvP.zip](https://github.com/thommcgrath/Beacon/files/2260125/Speed.Ark.PvP.zip)

**Comment by Tek4s** [comment]
okay I think I have found the problem, I have disabled many engramms or activated after I have these lines rausrated I have everything running again. I will set the unlocklevel to 500 so you will not get to the engrams.

Thank you very much for the help
